app/source/html/parts/UserMenuContents.html

```diff
@@ -20,7 +20,7 @@
     <a href="#continue-onboarding" data-bind="click: continueOnboarding"><i class="fa fa-fw ion-android-contact" aria-hidden="true"></i> Finish Profile</a>
 </li>
 <li data-bind="visible: !isAnonymous(), css: { 'is-disabled': isInOnboarding, active: active() === 'go-dashboard' }" class="go-dashboard">
-    <a data-bind="attr: { href: isInOnboarding() ? null : '/dashboard' }"><i class="fa fa-fw ion-stats-bars" aria-hidden="true"></i> Dashboard</a>
+    <a data-bind="attr: { href: isInOnboarding() ? null : '/dashboard' }"><i class="fa fa-fw ion-stats-bars" aria-hidden="true"></i> [[[Dashboard]]]</a>
```

Comment:
I almost sure we will need to use a different way to tag strings for translation at the client/app (the i18next module looks for a data-i18n attribute, and Knockout -for dynamic texts- for the text key in a binding or provided by the js viewmodel). Need to review this myself.

Reply:
Your right, I left this in by accident when i was testing if i18n worked on app files. sorry. I've reset this file